README.md

```diff
@@ -10,6 +10,14 @@ PowerShell Editor Services is a PowerShell module that provides common
 functionality needed to enable a consistent and robust PowerShell development
 experience in almost any editor or integrated development environment (IDE).
 
+## Front-ends using PSES
```

Comment:
"Front-ends" is probably more recognizable to folks but technically these are language service client apps, right?

Reply:
Yeah "LSP clients" is maybe better?

Reply:
That's a more "correct" description IMO. Whether that is meaningful to the average user who looks at this ... I dunno.

Reply:
Well I'm hoiping that users who get to the PSES README probably understand it somewhat